src/utils/common/blockchainMetrics/LatestBlocktime.ts

```diff
@@ -0,0 +1,10 @@
+//
+// Returns the most recent blocktime across a span of block objects
+//
+
+import { IBlock } from "types/NXTAPI";
+import { TimestampToDate } from "../Formatters";
+
+export function FetchLatestBlocktime(blocks: Array<IBlock>): string {
+  return TimestampToDate(blocks[0].timestamp);
```

Comment:
would it make sense to check blocks.length > 0 before blinding attempting to return it's timestamp value? is there a chance at runtime it's not set?

Reply:
Currently where it's called it would always have block data but that could change so I've made it throw an error if the blocks array has zero length.